examples/hosted-app-action-templates/typescript/build-action.js

```diff
@@ -0,0 +1,35 @@
+const esbuild = require('esbuild');
+const { resolve } = require('path');
+const yargs = require('yargs/yargs');
```

Comment:
Is this package already available?

Reply:
Yes, when I tested it didn't gave any error regarding this package to add it as a dev dependency